README.md

```diff
@@ -18,48 +18,30 @@ Autofill providers: give 3P autofill providers hooks that work seamlessly with n
 
-```javascript
-<form autocomplete="full-address" onautofill="autofillhandler">
-...
-</form>
-```
-
-`full-address` acts as an enhancement to existing behavior. Autofill can continue to use form input fields to preview and fill the fields when the user confirms their selection. Presence of full-address indicates that the browser should also provide a structured address object that the developer can process and use to implement own validation and fill behavior.
-
-![](https://screenshot.click/26-11-29h70-lrrrj.png)
-*Some browsers provide a preview state where selected address information is shown in the form but not yet visible to the page and script. This behavior is great and can remain as is.*
-
-With `full-address` site owners can eliminate the need for and use of hidden input fields for address. 
-
-## 2. Javascript handler for autofill
-In the HTML spec, the [autocomplete attribute fulfills two separate roles](https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls%3A-the-autocomplete-attribute): the spec talks about the autofill expectation mantle vs. the anchor mantle. In short, we rely on input fields to represent both the information that users would expect the browser to fill, as well as the place in which the browser would fill it (and to an extent, the UI in which users interact with the autofill browser feature). We believe it should be also possible to split these apart, to allow site owners with custom logic to bring their own validation and fill behaviors.
-
-We are proposing to introduce an event handler on the form element for autofill events to allow receiving autofilled values. This event would be the primary consumer of `full-address` object:
+We are proposing to introduce an event handler on the form element for autofill events to allow receiving autofilled values:
 ```javascript
 <script>
 const autofillhandler = event => {
-  const validated_values = validate(event.autofill_values());
-  updateAddress(validated_values);
-  event.preventDefault(); // stop browser autofill in favor of custom validation+fill logic
+  const validated_values = updateForm(event.autofill_values());
+  ... logic ...
 };
 </script>
 
 
-<form onautofill="autofillhandler" autocomplete="full-address">...</form>
+<form onautofill="autofillhandler">...</form>
 ```
 
-When the user commits to autofill, the form’s onautofill delegate is called with requested autocomplete data in structured format. At this point the developer can perform own validations and transforms on provided data, update the form, and fill it.
+When the user commits to autofill, the form’s onautofill delegate is called with requested autocomplete data in structured format. At this point the developer can perform their own validations and transforms on provided data and modify the form, before the control is passed back to the browser to perform the actual autofill.
```

Comment:
Non-blocking for this, but doesn't the "structured format" part require us to come up with some schema?
Alternatively, we could have the returned data be tied to the current form and be returned related to current field autocomplete attributes.
Another alternative could be for the browser to indicate (using autocomplete attribute values) the parts of the address that *don't* have an appropriate field.

We should probably spin up a separate issue for this..